docs/changelogs/v0.16.md

```diff
@@ -85,6 +86,33 @@ $ ipfs config Routing.Methods --json '{
 
 ```
 
+### 🌍 WebTransport new experimental Transport
+
+A new feature of [`go-libp2p`](https://github.com/libp2p/go-libp2p/releases/tag/v0.23.0) is [WebTransport](https://github.com/libp2p/go-libp2p/issues/1717).
+
+For now it is **disabled by default** and considered **experimental**.
+If you find issues running it please [report them to us](https://github.com/ipfs/kubo/issues/new).
+
+In the future we will change Kubo to listen on WebTransport by default (and listening) for anyone already listening `QUIC`, when we think it is stabilized.
+
+This transport will allow in-browser nodes to contact Kubo nodes using a new
```

Comment:
can’t do multiline-suggestion from my tablet…
```suggestion
WebTransport is a new transport protocol currently under development by the IETF and the W3C, and already implemented by Chrome. Conceptually, it’s like WebSocket run over QUIC instead of TCP. Most importantly, it allows browsers to establish (secure!) connections to WebTransport servers without the need for CA-signed certificates, thereby enabling any js-libp2p node running in a browser to connect to any kubo node, with zero manual configuration involved.
```

Reply:
Done